lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -129,6 +132,18 @@ define(['backbone', 'jquery', 'common/js/spec_helpers/ajax_helpers', 'common/js/
                 );
             };
 
+            createMockVerifiedTrackCohortsJson = function (enabled) {
+                if (enabled) {
+                    return {
+                        enabled: true,
+                        verified_cohort_name: "Verified Track"
```

Comment:
@cahrens Is this the cohort name suggested by Melanie or Julia? There's no mention of "track" and so there's some possibility of misunderstanding that the learners are in a verified state, rather than that they're on the verified track. Possible to make this "Verified Track Learners" instead?